src/app/app.component.html

```diff
@@ -7,11 +7,11 @@
 
         <!-- toggler button -->
         <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbarToggleExternalContent" aria-controls="navbarToggleExternalContent" aria-expanded="false" aria-label="Toggle navigation">
-          <span class="navbar-toggler-icon"></span>
-        </button>
+        <span class="navbar-toggler-icon"></span>
+      </button>
 
-        <!-- nav bar links -->
-        <div class="collapse navbar-collapse justify-content-end" id="navbarNav">
+        <!-- nav bar links if not login -->
+        <div class="collapse navbar-collapse justify-content-end" id="navbarNav" *ngIf="router.url != '/dashboard' && router.url != '/group'">
```

Comment:
Create a method in component using angular router service to detect the current url and return true/false accordingly